apps/auth/src/guards/jwt-auth.guard.ts

```diff
@@ -32,11 +36,18 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
         } catch (error) {
             if (error instanceof TokenExpiredError) {
                 const { userId } = jwtService.decode(token);
-                const user = await this.authService.regenerateToken(userId, response);
+                const user = this.usersService.getUserById({ id: userId });
```

Comment:
Do we need it here if we anyway getting user inside `validate` method of jwt.strategy?